apps/src/templates/progress/ProgressDetailToggle.story.jsx

```diff
@@ -53,7 +54,11 @@ export default storybook => {
     store.dispatch(setIsSummaryView(true));
     return {
       name: 'isSummary is true',
-      story: () => <ProgressDetailToggle store={store} />
+      story: () => (
+        <Provider store={store}>
```

Comment:
this does fix the error, but it obfuscates why this test was a problem -- it's currently testing the internals of our redux store, rather than just testing the component. i think we can make some small changes to fix this where, instead of setting up mock data that mimics the redux structure:

https://github.com/code-dot-org/code-dot-org/blob/4a95b3b34a4c12d24328c174003d293356e9e32d/apps/src/templates/progress/ProgressDetailToggle.story.jsx#L8-L50

we instead give the component the props we want it to have (e.g., most of that mock data is really just changing the `hasGroups` prop between true and false).

then, these tests would also better match our other storybook tests, as they currently look much different ([ShareAllowedDialog.story.jsx](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/ShareAllowedDialog.story.jsx) is a good example of a storybook entry that requires redux but uses props to change component structure instead of also testing redux internals).

so i think this test could look something like:

```javascript
storybook
  .storiesOf('Progress/ProgressDetailToggle', module)
  .withReduxStore({progress})
  .addStoryTable([
    {
      name: 'isSummary is true',
      story: () => (
        <ProgressDetailToggle isPlc={false} isSummaryView hasGroups={false} />
      )
    }
  ])
```

this would remove the need to dispatch redux actions and mock all the data within redux.

this is a really long comment 😅 so let me know if you want to chat more about this or if you have any questions!